ui/docs-app/docs_app/theme/assets/semantic-ui/js/docs_app/forms/deposit/NRDocumentValidationSchema.js

```diff
@@ -39,10 +36,7 @@ export const unique = (value, context, path, errorString) => {
 export const NRDocumentValidationSchema = Yup.object().shape({
   metadata: Yup.object().shape({
     // not sure but I assume it would be good idea to ask for a minimum length of title
```

Comment:
This comment could also be removed after the consensus that we cannot enforce any title length on users, other than that atleast some value is required

Reply:
@mirekys for sure, my bad